runtime/oti/shchelp.h

```diff
@@ -31,11 +31,12 @@
 extern "C" {
 #endif
 
-#define J9SH_MODLEVEL_JAVA5 1
-#define J9SH_MODLEVEL_JAVA6 2
-#define J9SH_MODLEVEL_JAVA7 3
-#define J9SH_MODLEVEL_JAVA8 4
-#define J9SH_MODLEVEL_JAVA9 5
+#define J9SH_MODLEVEL_JAVA5  1
```

Comment:
#836 is doing some cleanup of obsolete code. Can some of these old constants for JAVA5-JAVA7 be removed?

Reply:
@DanHeidinga pointed out cleaning up these old constants might affect IBM Java builds. I think we'd at least need a constant like J9SH_MODLEVEL_UNSUPPORTED. This type of change is better left for another commit.